mautic/mautic
# Add background-position and background-image to section content and wrapper

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This add background image position and all necessary fields to add image into content in Email builder and Landing builder.
Moreover this add Outlook tag for background-image when the builder is Email builder.

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/7211)
2. Clear cache and regenerate assets
3. Open Email Builder, Select a data-section element
4. Then you'll see additional field in Customize Section menu to change the background image position, and all field necessary to add image into content. Input some value to it, then save.

![screenshot_2019-02-04 editer l email - double opt-in webmecanik automation](https://user-images.githubusercontent.com/27768270/52205530-a1115e00-2877-11e9-83f8-04d2f5862ec5.png)
